changelog.d/20250721_122944_chris_sc_42250.rst

```diff
@@ -0,0 +1,24 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- With this release, we are moving from the existing "Multi-User Endpoint" /
+  "Single-User Endpoint" dichotomy to a unified model where every endpoint
+  supports configuration templates.
+
+Deprecated
+^^^^^^^^^^
+
+- The ``multi_user`` config flag is no longer needed when every endpoint is
+  template capable; the flag has been deprecated.
+
+Changed
+^^^^^^^
+
+- When configuring an endpoint, the command line automatically decides whether to
+  initialize identity mapping based on whether or not the running user is privileged.
+  This can be overridden with the following flags:
+
+  - ``--identity-mapping``: Force identity mapping to be enabled, even if the user is
+    not privileged.
+  - ``--no-identity-mapping``: Force identity mapping to be disabled, even if the user
+    is privileged.
```

Comment:
You might also note that even if identity is force-configured, that it won't be enabled if the _running_ user doesn't have privileges.  (Vice-versa is also true; even if force-not-configured, if the running user has privileges, we all-but require identity mapping.)

Reply:
Per verbal conversation, though identity mapping is a critical tool by which we do multi-user, the concept -- and the concept that we have publicly shared and advertised -- is multi-user.  We are sticking with `--multi-user` and `--no-multi-user`, in agreement with the ticket.